controller/cache/cache.go

```diff
@@ -102,6 +102,12 @@ var (
 	clusterCacheRetryUseBackoff bool = false
 )
 
+// Constants for EndpointSlices
+const (
+	serviceDiscoveryGroup             = "discovery.k8s.io"
+	serviceDiscoveryEndpointSliceKind = "EndpointSlice"
```

Comment:
It is already possible to configure which resources update to ignore with customizations. https://argo-cd.readthedocs.io/en/stable/operator-manual/reconcile/ . Have you tried to use this feature?

Additionally, if you do not manage EndpointSlice manually, you can fully omit them from the watched resources. https://argo-cd.readthedocs.io/en/stable/operator-manual/declarative-setup/#resource-exclusioninclusion

Reply:
I did that exclusion via resource updates rule. However, I think it's something everyone should do, thus having them hard-coded seems beneficial.